sdk/batch/batch/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Release History
+
+## 10.1.0 (2022-02-09)
+
+### Features
+
+- Added property `UploadHeaders` to `OutputFileBlobContainerDestination`.
+  - Allows users to set custom HTTP headers on resource file uploads.
+  - Array of type HttpHeader (also being added).
+- Added boolean property `allowTaskPreemption` to `JobSpecification`, `CloudJob`, `JobAddParameter`, `JobPatchParameter`, `JobUpdateParameter`
+  - Mark Tasks as preemptible for higher priority Tasks (requires Comms-Enabled or Single Tenant Pool).
+- Replaced comment (title, description, etc.) references of "low-priority" with "Spot/Low-Priority", to reflect new service behavior.
```

Comment:
I know you probably got this from the C# changelog, but IMO we don't need to document a comment change here.